NDLANO/article-converter
# Add correct id and props

To test: 
- Run article-converter with ndla-frontend from branch "implement-new-clickToggle"
- Check that all clickToggles work (in authorByLine, FigureLicense etc)
